.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/go/src/github.com/nec-blockchain/minbft
+    docker:
+      - image: circleci/buildpack-deps:xenial
+    environment:
+      SGX_SDK: /opt/intel/sgxsdk
+    steps:
+      - run:
+          name: Install Go
+          command: |
+            wget "https://golang.org/dl/go1.9.4.linux-amd64.tar.gz" -O golang.tar.gz
+            sudo tar -C /usr/local -xaf golang.tar.gz
+            rm golang.tar.gz
+            echo 'export GOPATH=$HOME/go' >> $BASH_ENV
+            echo 'export PATH=$GOPATH/bin:/usr/local/go/bin:$PATH' >> $BASH_ENV
+      - run:
+          name: Install Intel SGX SDK
+          command: |
+            sudo apt-get update -q
+            sudo apt-get install -qy --no-install-recommends ca-certificates build-essential python wget git make pkg-config
+            wget "https://download.01.org/intel-sgx/linux-2.1.1/ubuntu64-server/sgx_linux_x64_sdk_2.1.101.42529.bin" -O sgx_installer.bin
+            chmod +x sgx_installer.bin
+            echo -e "no\n$(dirname $SGX_SDK)" | sudo ./sgx_installer.bin
+            rm sgx_installer.bin
+            echo '. $SGX_SDK/environment' >> $BASH_ENV
+            echo 'export PATH=$PATH:$SGX_SDK/bin:$SGX_SDK/bin/x64' >> $BASH_ENV
+            echo 'export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$SGX_SDK/pkgconfig' >> $BASH_ENV
+            echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$SGX_SDK/sdk_libs' >> $BASH_ENV
```

Comment:
`. $SGX_SDK/environment` should update `PATH`, `PKG_CONFIG_PATH`, and `LD_LIBRARY_PATH` environment variables accordingly. Do we really need to duplicate that in lines 28-30?